seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/TaskExecutionService.java

```diff
@@ -252,9 +259,25 @@ public TaskDeployState deployTask(@NonNull TaskGroupImmutableInformation taskImm
                         taskImmutableInfo.getExecutionId()));
         TaskGroup taskGroup = null;
         try {
+            Set<ConnectorJarIdentifier> connectorJarIdentifiers =
+                    taskImmutableInfo.getConnectorJarIdentifiers();
             Set<URL> jars = taskImmutableInfo.getJars();
             ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
-            if (!CollectionUtils.isEmpty(jars)) {
+            if (!CollectionUtils.isEmpty(connectorJarIdentifiers)) {
+                // Prioritize obtaining the jar package file required for the current task execution
+                // from the local,
+                // if it does not exist locally, it will be downloaded from the master node.
```

Comment:
```suggestion
                // Prioritize obtaining the jar package file required for the current task execution
                // from the local, if it does not exist locally, it will be downloaded from the master node.
```